trino/client.py

```diff
@@ -1191,31 +1305,8 @@ def _create_value_mapper(self, column) -> ValueMapper:
         else:
             return NoOpValueMapper()
 
-    def _get_precision(self, column: Dict[str, Any]):
+    def _get_precision(self, column: Dict[str, Any]) -> int:
         args = column['arguments']
         if len(args) == 0:
             return 3
         return args[0]['value']
-
-
-class RowMapper:
```

Comment:
why moved?

Reply:
extract just the movement to separate commit with reasoning in the msg

Reply:
Reverted. Used [Postponed Evaluation of Annotations](https://peps.python.org/pep-0563/) instead of reordering class definition. Reason was that type hint contained names that have not been defined yet.